lib/farscape/transition.rb

```diff
@@ -28,6 +29,12 @@ def invoke(args={})
     end
 
     def method_missing(meth, *args, &block)
+      if [:omitting, :using].include?(meth)
```

Comment:
I'd rather not define statically-named methods using method_missing, it breaks stuff like `respond_to`, `methods`, and doc generation. If you want to DRY these methods, you can either use a delegation library like [active_support's](http://guides.rubyonrails.org/active_support_core_extensions.html#method-delegation) or use `define_method`.